tests/functionaltests/suites/harvesting/data/.gitignore

```diff
@@ -0,0 +1,30 @@
+# This file exists to force git to include this directory in the code
```

Comment:
I'd actually prefer if the other (what other `gitkeep` files are there elsewhere in the codebase if not in the tests directories) files would be renamed to `.gitignore` and would have a textual description of their purpose, as I've done here.

It took me a while to understand what this `.gitkeep` thing was. It is not a git feature and is therefore not documented in the git docs. It is merely a convention that these files are called `.gitkeep`. IMHO using `.gitignore` with a brief explanation of the file's purpose inside is more explicit.